py-polars/polars/utils.py

```diff
@@ -8,7 +8,8 @@
 import sys
 import warnings
 from collections.abc import MappingView, Reversible, Sized
-from datetime import date, datetime, time, timedelta, timezone, tzinfo
+from datetime import date, datetime, time, timedelta, timezone
+from datetime import tzinfo as tzinfo_t
```

Comment:
Why this alias?

Reply:
because below `tzinfo` is used as a variable name, so `mypy` confuses them

the alternative would be to rename the variable...that might be clearer actually, thanks